minion/plugins/wsman-plugin/src/main/java/org/opennms/lokahi/minion/wsman/WsmanDetector.java

```diff
@@ -0,0 +1,150 @@
+
+        try {
+            if (!config.is(WsmanDetectorRequest.class)) {
+                throw new IllegalArgumentException(
+                        "config must be an WsmanDetectorRequest; type-url=" + config.getTypeUrl());
+            }
+
+            WsmanDetectorRequest wsmanDetectorRequest = config.unpack(WsmanDetectorRequest.class);
+
+            CompletableFuture<ServiceResult> future = getAsync(wsmanDetectorRequest, LOG)
+                    .thenApply(serviceResult -> {
+                        ServiceResult serviceResponse = ServiceResult.newBuilder()
+                                .setStatus(serviceResult.getStatus())
+                                .setIpAddress(serviceResult.getIpAddress())
+                                .setService(serviceResult.getService())
+                                .build();
+                        return serviceResponse;
+                    });
+
+            return future;
+
+        } catch (IllegalArgumentException e) {
+            LOG.debug("Invalid WSMAN Criteria during detection of interface {}", host, e);
+            System.out.println(">>> IllegalArgumentException: " + e.toString());
+            return CompletableFuture.completedFuture(getErrorResult(host));
+        } catch (Exception e) {
+            LOG.debug("Unexpected exception during WSMAN detection of interface {}", host, e);
+            System.out.println(">>> Exception: " + e.toString());
+            return CompletableFuture.completedFuture(getErrorResult(host));
+        }
+    }
+
+    public CompletableFuture<ServiceResult> getAsync(WsmanDetectorRequest wsmanDetectorRequest, Logger LOG) {
+
+        return CompletableFuture.supplyAsync(() -> {
+            boolean serviceStatus = false;
+
+            try {
+                WsmanConfiguration wsmanConfiguration = wsmanDetectorRequest.getAgentConfiguration();
+                WSManEndpoint endpoint;
+                InetAddress address = InetAddress.getByName(wsmanConfiguration.getHost());
+
+                endpoint = WsmanEndpointUtils.fromWsmanDetectorRequest(address, wsmanDetectorRequest);
+                WSManClientFactory m_factory = new CXFWSManClientFactory();
+                WSManClient client = m_factory.getClient(endpoint);
+
+                Identity identity = null;
+                final Map<String, String> attributes = new HashMap<>();
+                try {
+                    identity = client.identify();
+
+                } catch (WSManException e) {
+                    LOG.info("Identify failed for endpoint {} with error {}.", endpoint, e);
+                    System.out.println(">>> WSManException: " + e.toString());
+                }
+
+                if (identity != null) {
+                    serviceStatus = true;
+                    LOG.info("\n\n>> UP, Node is UP, Congrats, WSMAN identified/ detected successfully. {} ", endpoint);
+                } else LOG.info(">> Node DOWN, not able to detect {} ", endpoint);
```

Comment:
```suggestion
                } else {
                    LOG.info(" Node DOWN, not able to detect {} ", endpoint);
                }
```
remove >> in the logs.

Reply:
>> signs has been removed